exercises/30_functions/New_excercises_ideas/group_1_level_5_Vorschlag_5.ipynb

```diff
@@ -1,12 +1,10 @@
 {
```

Comment:
die Datei und das verzeichnis bitte löschen. Neue Ideen wie diese Datei dann in zukunf in einem eigenen Branch commiten.